Sources/Apollo/ApolloClient.swift

```diff
@@ -13,6 +13,8 @@ public enum CachePolicy {
   case returnCacheDataDontFetch
   /// Return data from the cache if available, and always fetch results from the server.
   case returnCacheDataAndFetch
+  /// Attempt to fetch from the server, using cache data if available on error
+  case fetchReturningCacheDataOnError
```

Comment:
How do we handle the  `TTL` with this cache policy?

Reply:
I didn't consider TTL because that was still in the other branch and I didn't want to involve it yet until it was merged.

TTL shouldn't change much from this: If the cache is out of date, it'll return a `CacheError.dataExpired` error if the fetch also error'd out.